docs/src/verbosity.md

```diff
@@ -0,0 +1,9 @@
+# Controlling Solver Verbosity
```

Comment:
DiffEqDocs needs a PR that explains it in more detail how it's a solver-package specific thing.